src/Channels.Networking.Libuv/UvTcpConnection.cs

```diff
@@ -37,24 +37,24 @@ public UvTcpConnection(UvThread thread, UvTcpHandle handle)
             _sendingTask = ProcessWrites();
         }
 
-        public IWritableChannel Output => _output;
+        public IWritableChannel Output => _output.Output;
 
-        public IReadableChannel Input => _input;
+        public IReadableChannel Input => _input.Input;
 
         private async Task ProcessWrites()
         {
             try
             {
                 while (true)
                 {
-                    var buffer = await _output.ReadAsync();
+                    var buffer = await _output.Input.ReadAsync();
```

Comment:
Shouldn't this be using a property? or have some renaming? Something about it jars